proteus/NonlinearSolvers.py

```diff
@@ -617,6 +618,102 @@ def solve(self,u,r=None,b=None,par_u=None,par_r=None):
         log("   Newton it %d norm(r) = %12.5e  \t\t norm(r)/(rtol*norm(r0)+atol) = %12.5e"
             % (self.its,self.norm_r,(self.norm_r/(self.rtol_r*self.norm_r0+self.atol_r))),level=1)
 
+class POD_Newton(Newton):
+    """Newton's method on the reduced order system based on POD"""
+    def __init__(self,
+                 linearSolver,
+                 F,J=None,du=None,par_du=None,
+                 rtol_r  = 1.0e-4,
+                 atol_r  = 1.0e-16,
+                 rtol_du = 1.0e-4,
+                 atol_du = 1.0e-16,
+                 maxIts  = 100,
+                 norm = l2Norm,
+                 convergenceTest = 'r',
+                 computeRates = True,
+                 printInfo = True,
+                 fullNewton=True,
+                 directSolver=False,
+                 EWtol=True,
+                 maxLSits = 100):
+        Newton.__init__(self,
+                linearSolver,
+                F,J,du,par_du,
+                rtol_r,
+                atol_r,
+                rtol_du,
+                atol_du,
+                maxIts,
+                norm,
+                convergenceTest,
+                computeRates,
+                printInfo,
+                fullNewton,
+                directSolver,
+                EWtol,
+                maxLSits)
+        self.DB = 11
+        U = np.loadtxt('SVD_basis')
```

Comment:
@mfarthin, if  I generate the SVD_basis  for  both u and F in svd_burgers, then here I just read them both in and call `deim_utils.get_deim` to construct `rho` and  your `PF`.